src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -973,22 +973,33 @@ public function getDirty()
     }
 
     /**
-     * Determine if the new and old values for a given key are numerically equivalent.
+     * Determine if the new and old values for a given key are equivalent.
      *
      * @param  string  $key
      * @return bool
      */
-    protected function originalIsNumericallyEquivalent($key)
```

Comment:
Don't make sense override this function in a model, because, this should not only check if is numerically equivalent, but to check if is equivalent